packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -40,6 +55,84 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
         return providers
       }
     },
+    issues: {
+      type: new GraphQLNonNull(JiraServerIssueConnection),
+      description:
+        'A list of issues coming straight from the jira integration for a specific team member',
+      args: {
+        first: {
+          type: GraphQLInt,
+          defaultValue: 100
+        },
+        after: {
+          type: GraphQLISO8601Type,
+          description: 'the datetime cursor'
+        },
+        queryString: {
+          type: GraphQLString,
+          description: 'A string of text to search for, or JQL if isJQL is true'
+        },
+        isJQL: {
+          type: new GraphQLNonNull(GraphQLBoolean),
+          description: 'true if the queryString is JQL, else false'
+        },
+        projectKeyFilters: {
+          type: new GraphQLList(new GraphQLNonNull(GraphQLID)),
+          descrption:
+            'A list of projects to restrict the search to. format is cloudId:projectKey. If null, will search all'
+        }
+      },
+      resolve: async (
+        {teamId, userId},
+        // {first, queryString, isJQL, projectKeyFilters},
+        {first},
+        {authToken, dataLoader}: GQLContext
+      ) => {
+        const viewerId = getUserId(authToken)
+        if (viewerId !== userId) {
+          const err = new Error('Cannot access another team members issues')
+          standardError(err, {tags: {teamId, userId}, userId: viewerId})
+          return connectionFromTasks([], 0, err)
+        }
+
+        const integrationManager = await TaskIntegrationManagerFactory.initManager(
```

Comment:
-1 let's call `new JiraServerTaskIntegrationManager` directly because it has methods on it that this interface does not. that way we don't have to use optional chaining below when we know already know what it is